x/ibc/07-tendermint/evidence.go

```diff
@@ -24,11 +24,11 @@ var (
 // Evidence is a wrapper over tendermint's DuplicateVoteEvidence
 // that implements Evidence interface expected by ICS-02
 type Evidence struct {
-	ClientID         string                `json:"client_id" yaml:"client_id"`
+	ClientID         string                `json:"client_id" yaml:"client_id"` // ID of the client commiting the misbehaviour
 	FromValidatorSet *tmtypes.ValidatorSet `json:"from_validator_set" yaml:"from_validator_set"`
 	Header1          Header                `json:"header1" yaml:"header1"`
 	Header2          Header                `json:"header2" yaml:"header2"`
-	ChainID          string                `json:"chain_id" yaml:"chain_id"`
+	ChainID          string                `json:"chain_id" yaml:"chain_id"` // Id of the misbehaving chain
```

Comment:
Isn't this implied by the `ClientID`? Does the user need to pass it in?

Reply:
yes, this is the same logic as before. I just added the comment here

Reply:
I guess what I'm saying here is that we don't need the chainID here because we should be able to read from the client ID

Reply:
the client doesn't store the chain ID tho